scripts/verify_consistency.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# bash scripts/verify.sh OBJECT_TYPE CUTOFF_TIME
+# OBJECT_TYPE is `files`, `bundles`, or `blobs`
+# CUTOFF_TIME is lower limit as an iso8601 datetime
+set -eu
+
+if [[ "$#" -ne 2 ]]; then
+    echo "./$0 OBJECT_TYPE CUTOFF_TIME"
+    echo "OBJECT_TYPE is one of 'files', 'bundles', or 'blobs'"
+    echo "CUTOFF_TIME determines which objects are selected for review and"
+    echo "    is formatted as an iso8601 datetime. To verify all present"
+    echo "    objects, provide a CUTOFF_TIME of zero (or something else"
+    echo "    absurdly small)"
+    exit 1
+fi
+
+OBJECT_TYPE="$1"
+CUTOFF_TIME="'$2'"
+SECRET_NAME="dcp/dss/${DSS_DEPLOYMENT_STAGE}/${OBJECT_TYPE}_last_verified"
+
+# We want to get a list of all objects of prefix $TYPE that were created after
+# $CUTOFF. To do that, we have to list all objects in $DSS_S3_BUCKET and sort
+# them ourselves, locally.
+aws s3api list-objects-v2 \
+    --bucket "$DSS_S3_BUCKET" \
```

Comment:
Since this only checks `s3`, if there's a google file that exists in the `gcp` bucket, but not in the `s3` bucket, will this catch it?